### ISSUE TITLE
ion-select changes not appearing on page for model-driven form with change detection set to OnPush

### ISSUE THREAD

**Issue by fiznool** [ionic-team/ionic#9205]
#### Short description of the problem:

`<ion-select>` changes are not appearing when using a FormBuilder form on a page, when `ChangeDetectionStrategy.OnPush` is used on a component.

#### What behavior are you expecting?

`<ion-select>` changes should appear when using a FormBuilder form on a page, when `ChangeDetectionStrategy.OnPush` is used on a component.

**Steps to reproduce:**
1. Add a component which uses FormBuilder, with an `ion-select`.
2. Set the change detection strategy to OnPush.
3. Try to select an item from the select box.

The plunker below best illustrates the issue.

**Which Ionic Version?** 1.x or 2.x

Ionic 2 RC2

#### Plunker that shows an example of your issue

http://plnkr.co/edit/SUhomP6GZ687mvzjA9jd?p=preview

The first three examples show the select box updating correctly after an item is selected, but the 4th does not.

Notice that ngModel-based forms are not affected, only ones built using FormBuilder.

**Run `ionic info` from terminal/cmd prompt:** (paste output below)

```
Your system information:

Cordova CLI: 6.3.1 
Ionic Framework Version: 2.0.0-rc.2
Ionic CLI Version: 2.1.8
Ionic App Lib Version: 2.1.4
Ionic App Scripts Version: 0.0.44
ios-deploy version: Not installed
ios-sim version: Not installed
OS: Linux 4.8
Node Version: v6.7.0
Xcode version: Not installed
```

**Comment by iron9light** [comment]
Have same issue with Range component.
Ionic 2.0.0-rc.3
https://github.com/iron9light/ionic-bug/tree/master/RangeNotUpdateWhenValueChangedByClick

**Comment by gnujeremie** [comment]
Still no fix, ionic 2.1.0 :(
